lib/bolt/outputter/human.rb

```diff
@@ -331,10 +331,29 @@ def print_module_list(module_list)
         end
       end
 
-      def print_targets(targets)
-        count = "#{targets.count} target#{'s' unless targets.count == 1}"
-        @stream.puts targets.map(&:name).join("\n")
-        @stream.puts colorize(:green, count)
+      def print_targets(target_list, inventoryfile)
+        inventory = colorize(:cyan, inventoryfile.to_s)
+        adhoc     = colorize(:cyan, "Not found in inventory file")
+
+        targets  = []
+        targets += target_list[:inventory].map { |target| [inventory, target.name] }
+        targets += target_list[:adhoc].map { |target| [adhoc, target.name] }
+
+        if targets.any?
```

Comment:
Given that there are only two values for the first column and we already group the targets by them, it seems redundant to print that column every time. Would it make more sense for  it to be a header? Or maybe just an annotation for targets that _don't_ exist in the inventory (and nothing for the ones that do, since we report the inventory file below)?